internal/utils/browser.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"github.com/pkg/browser"
+)
+
+// BrowserInterface provides access to browser
+type BrowserInterface interface {
+	Open(url string) error
+}
+
+// Browser is a mocked browser
+type Browser struct{}
```

Comment:
[style/nit]

naming an interface with `Interface` as the suffix (or `I` as the prefix... ugh!) is something worth avoiding (it never adds any _real_ meaning to the name other than heft).

while the caveat of "its always best to just start with the `struct` and not an `interface`" holds true everywhere, I think I see where you're going with the interface here for testing/mocking purposes

with that in mind, a fairly typical pattern for this (to help avoid the `Interface` naming suffix would be:

```go
type MyThing interface { ... }

func NewMyThing(...) MyThing {
  return myThing{...}
}

// myThing doesn't need to be exported b/c it inherently implements MyThing
// which is the only type we're really exporting from this package (and the constructor)
type myThing struct { ... }
```